activiti-spring-boot-starter/src/test/resources/processes/ProcessWithIntermediateThrowMessage-extensions.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "intermediateThrowMessageEvent",
+  "name": "intermediateThrowMessageEvent.bpmn20.xml",
+  "extensions": {
+	    "properties": {
+	      "process_variable_name": {
+	        "id": "process_variable_id",
+	        "name": "process_variable_name",
+	        "type": "string",
+	        "value": "value"
+	      }
+	    },
+	    "mappings": {
+	    	"IntermediateThrowEvent_0la4q0b": {
+		      "inputs": {
```

Comment:
Indentation is using tabs instead of spaces.

Reply:
Done. See commit 3bcd8cfc730eb662623ff2ea9a7025066827ada1